versions/v0.1.5/Specification.MD

```diff
@@ -601,8 +603,8 @@ _Utils_
     * OutputType - [any]
     * Output - single string adds output to that variable, list of strings applies output of each input object to the corresponding outputlist item
 
-_Custom Operations_:
-* Language specific method for defining a custom operation (i.e. in go using go get)
+_Additional Operations_:
+* If further operations are needed.  Please suggest them through the Github Issue or PR  system.
```

Comment:
Not sure this statement should be part of the specification... Perhaps a better place would be a CONTRIBUTING.md in the root of the repo?

Reply:
I have made a CONTRIBUTING.md with a quick blurb we can improve in the future.